belt/src/services/utils.ts

```diff
@@ -9,10 +9,12 @@ import * as config from './config'
 export function getContractDirs(conf: config.App) {
   const contractDirs = ls(conf.contractsDir)
 
-  return contractDirs.map((d) => ({
-    dir: d,
-    version: conf.compilerSettings.versions[d],
-  }))
+  return contractDirs
+    .map((d) => ({
+      dir: d,
+      version: conf.compilerSettings.versions[d],
+    }))
+    .filter((p) => !!p.version)
```

Comment:
When will there not be a version?

Reply:
We have 5 different versions of contracts: v4, v5, v6, v6-ovm, v7
We have 2 belt config files:

- `app.config.json`, defines 4 different versions, compiled with `solc`
- `app.config.ovm.json`, defines 1 version, compiled with `solc-ovm`

So basically we just compile the versions defined in the config file, vs. as before all version directories in `contractsDir`.